resources/views/users/dashboard/my-events-create.antlers.html

```diff
@@ -207,13 +176,13 @@
                 {{ partial:event-submission-form/text
```

Comment:
I can't comment on the right section. But I noticed if there are errors with the submission the image gets lost

Reply:
Losing the image is normal behaviour - you can't pass an `old()` value into a file input (I think it's a browser thing) - there's a bunch of security concerns with it)